cypress/fixtures/ledgerActions/bitcoin-broadcast-signed-transaction.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "bitcoin-broadcast-signed-transaction",
+  "payload": {
+    "hex": "raw_transaction",
+    "network": "1",
```

Comment:
📣 should be `mainnet` or `testnet` or `regtest`

Reply:
It is just the fixtures though :man-shrugging:

Reply:
cool